src/Schema/Grammar.php

```diff
@@ -48,6 +48,14 @@ public function compileTableExists()
         return 'select * from information_schema.tables where table_schema = \'\' and table_name = ?';
     }
 
+    /**
+     * @return string
+     */
+    public function compileGetAllTables()
+    {
+        return 'select * from information_schema.tables where table_schema = \'\'';
```

Comment:
This SQL includes view table.  

I think it should be to add an additional table_type condition to exclude it as well as [MySqlGrammer::compileGetAllTables()](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php#L60-L68).

https://cloud.google.com/spanner/docs/information-schema#tables

```suggestion
        return 'select * from information_schema.tables where table_schema = \'\' and table_type = \'BASE TABLE\'';
```